### DISCUSSION TITLE
🖋: ToDoアプリの作成でgitを使うか

### DISCUSSION THREAD

**Discussion by ghost** [ws-4020/mobile-app-crib-notes#106] [Ideas]
ToDoアプリの作成でgitのinitやcommitについて触れていないが、ドキュメントの手順にあったほうがいいかもしれない

**Comment by tani-takanori** [comment]
コンテンツにて学習した方から
> コード修正量が多いところは差分表示ではなかったのですが、多少多くても差分表示されてる方が理解の助けになりそうだなぁと思いました。

というFeedbackあり
